researchstudio-sat/webofneeds
# Request Contact Without Need + Faster SPARQL Query

Test: 
1. Cause the creation of a need owned by the debug bot (or work with two accounts and a matcher)
2. go to the post-info page of that bot-owned need (e.g. by clicking the avatar in an open conversation)
3. once logged out and once logged in (with a different account in the future than the one to trigger the bot, atm the same account works too) do the following:
    1. click the "request contact button"
    2. write a message
    2. if you're working with two accounts, accept with the second account
    3. in the now open conversation check if you can see your message
